src/BehaviorsSDKManaged/Microsoft.Xaml.Interactions/Core/NavigateToPageAction.cs

```diff
@@ -134,13 +133,29 @@ public object Execute(object sender, object parameter)
 
             Frame frame = navigateElement as Frame;
 
+            // Temporary work around for Uno until it implements IXamlMetadataProvider
+#if __UWP__
+            var pageType = xamlType.UnderlyingType;
+#else
+            Type pageType = null;
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+            for (int i = 0; (i < assemblies.Length) && (pageType == null); i++)
+            {
+                pageType = assemblies[i].GetType(this.TargetPage, false);
+            }
```

Comment:
```suggestion
```suggestion
      var pageType  = AppDomain.CurrentDomain.GetAssemblies()
         .Select(a => a.GetType(this.TargetPage, false))
         .FirstOrDefault(t => t != null);
```
```

Reply:
Fixed